frontend/src/app/visualization/wordcloud/wordcloud.component.ts

```diff
@@ -77,6 +77,10 @@ export class WordcloudComponent implements OnChanges, OnInit, OnDestroy {
         }
     }
 
+    ngOnDestroy(): void {
+        this.chart.destroy();
```

Comment:
The word cloud tasks aren't aborted now anymore. Or is this done without Celery now?